client/src/index.jsx

```diff
@@ -1,30 +1,74 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Calendar from './components/Calendar.jsx';
-import Time from './components/Time.jsx';
-import People from './components/People.jsx';
+import axios from 'axios';
+import $ from 'jquery';
+import Reservation from './components/Reservation/Reservation.jsx';
+import ReservationTop from './components/Reservation/ReservationTop.jsx';
+import Order from './components/Order/Order.jsx';
+import BusinessInfo from './components/BusinessInfo/BusinessInfo.jsx';
+import Hours from './components/Hours/Hours.jsx';
+
+import style1 from '../style/style.css';
+import style2 from '../style/calendar.css';
+import style3 from '../style/modal.css';
+import style4 from '../style/table.css';
+import style5 from '../style/sticky.css';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      restaurant: null,
+      resId: 1,
     };
   }
 
+  componentDidMount() {
+    const { resId } = this.state;
+    axios.get(`http://127.0.0.1:5555/all/${resId}`)
+      .then((data) => {
+        this.setState({
+          restaurant: (data.data)[0],
+        });
+        console.log('DATA: ', data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  topsticker() {
+    const header = $(".reservations-sticky-bar");
+    $(window).scroll(() => {
+      const scroll = $(window).scrollTop();
+
+      if (scroll >= 240) {
+        header.removeClass('hidden');
+      } else {
+        header.addClass('hidden');
+      }
+    });
+  }
+
```

Comment:
I really like the functionality of your reservations top-bar. But, mixing in jQuery is bad here.